modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/shelf/MenuShelfItems.kt

```diff
@@ -32,6 +35,7 @@ fun MenuShelfItems(
     shelfViewModel: ShelfViewModel,
     normalBackgroundColor: Color = Color.Transparent,
     selectedBackgroundColor: Color = Color.Black,
+    navigationBarPadding: Boolean = false,
```

Comment:
It didn't feel very clean passing in if the navigation bar padding was included but I could think of a better way.